Civi/Api4/Query/Api4SelectQuery.php

```diff
@@ -592,8 +592,13 @@ protected function createSQLClause($fieldAlias, $operator, $value, $field, int $
       }
       return $sql ? implode(' AND ', $sql) : NULL;
     }
+
+    // The CONTAINS operator matches a substring for strings. For arrays & serialized fields,
+    // it only matches a complete (not partial) string within the array.
```

Comment:
Maybe note that we don't match complete strings for PHP serialized?

Reply:
I'll be there's a way to support that type of serialization too! Probably something like:
```
        case \CRM_Core_DAO::SERIALIZE_PHP:
          $operator = 'LIKE';
          $value = '%"' . serialize($value . '"%';
          break;
```
But I haven't tested that. Best to merge this PR first.

Reply:
Sure, I assumed it was a don't bother to support it because there aren't many of them situation.